frontend/README.md

```diff
@@ -0,0 +1,13 @@
+Tobira Frontend
+==============
+
+
+Building the Tobira Frontend
+---------------------------
+
+Use NPM to get all necessary dependencies and to build the Tobira frontend:
+
+```sh
+npm ci
```

Comment:
Not sure if `npm ci` is recommended for local development. In that case, `npm install` should be preferred, I think.

Reply:
I don't particularly care but `npm install` will regularly cause modifications to the `package-lock.json` which means that you need to be more careful about what you commit.

Reply:
Ok, from [here](https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci): 

> Use `npm ci` if you need a deterministic, repeatable build. For example during continuous integration, automated jobs, etc. and **when installing dependencies for the first time**, instead of `npm install`.

So yeah, TIL. Let's keep `npm ci`.